build.zig

```diff
@@ -97,6 +97,8 @@ fn addLinkerFlags(b: *Build, webui: *Compile, enable_tls: bool) !void {
     } else if (is_windows) {
         webui.linkSystemLibrary("ws2_32");
         webui.linkSystemLibrary("ole32");
+		webui.linkSystemLibrary("dwmapi");
+		webui.linkSystemLibrary("gdi32");
```

Comment:
Should follow indentation above

Reply:
I missed it.
I use tab on my editor, github's default tab size is 8.
Can I add configuration file for github to set 'tab size 4'?